Mage/src/main/java/mage/game/permanent/token/ImpToken.java

```diff
@@ -0,0 +1,36 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesSourceTriggeredAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.DamagePlayersEffect;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.TargetController;
+
+public final class ImpToken extends TokenImpl {
+
+    public ImpToken() {
+        super("Imp Token", "2/2 red Imp creature token with \"When this creature dies, it deals 2 damage to each opponent.\"");
+        cardType.add(CardType.CREATURE);
+        subtype.add(SubType.IMP);
+        color.setRed(true);
+        power = new MageInt(2);
+        toughness = new MageInt(2);
+
+        // When this creature dies, it deals 2 damage to each opponent.
+        Effect effect = new DamagePlayersEffect(2, TargetController.OPPONENT);
+        effect.setText("it deals 2 damage to each opponent");
```

Comment:
(I'm pretty sure rules text generation can handle this without manually setting?)